_data/kit_versions.yml

```diff
@@ -1,2 +1,4 @@
+- version: 2022.0.1
+  release: 2021-11-20
```

Comment:
```suggestion
  released: 2021-11-20
```

Reply:
Good spot! We should consider validating this file in CI.